ui-tests/tools/com.liferay.ide.ui.fragment.tests/src/com/liferay/ide/ui/fragment/tests/NewFragmentFilesWizardMavenTests.java

```diff
@@ -16,24 +16,331 @@
 
 import com.liferay.ide.ui.liferay.SwtbotBase;
 
+import java.io.IOException;
+
+import org.eclipse.core.runtime.IPath;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 /**
  * @author Vicky Wang
  * @author Sunny Shi
+ * @author Lily Li
  */
 public class NewFragmentFilesWizardMavenTests extends SwtbotBase {
 
+	@BeforeClass
+	public static void addLiferayServer() throws IOException {
+		envAction.unzipServer();
+
+		dialogAction.openPreferencesDialog();
+
+		dialogAction.openServerRuntimeEnvironmentsDialogTry();
+
+		dialogAction.openNewRuntimeWizard();
+
+		wizardAction.prepareLiferay7RuntimeType();
+
+		wizardAction.next();
+
+		IPath serverDir = envAction.getLiferayServerDir();
+
+		IPath fullServerDir = serverDir.append(envAction.getLiferayPluginServerName());
+
+		wizardAction.prepareLiferay7RuntimeInfo(_serverName, fullServerDir.toOSString());
+
+		wizardAction.finish();
+
+		dialogAction.confirmPreferences();
+
+		wizardAction.openNewLiferayServerWizard();
+
+		wizardAction.prepareNewServer(_serverName);
+
+		wizardAction.finish();
+	}
+
+	@AfterClass
+	public static void deleteRuntime() {
+		dialogAction.openPreferencesDialog();
+
+		dialogAction.openServerRuntimeEnvironmentsDialogTry();
+
+		dialogAction.deleteRuntimeTryConfirm(_serverName);
+
+		dialogAction.confirmPreferences();
+	}
+
 	@Test
-	public void addFragmentJsp() {
+	public void addFragmentFilesJsp() {
+		String projectName = "test-fragment-files-jsp-maven";
+
+		wizardAction.openNewFragmentWizard();
+
+		wizardAction.prepareFragmentMaven(projectName);
+
+		wizardAction.next();
+
+		wizardAction.openBrowseOsgiBundleDialog();
+
+		dialogAction.prepareText("com.liferay.comment.web");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		viewAction.openNewFragmentFileWizard();
```

Comment:
I'll ask Rui help change the name of this method. Might take some effects on your code. Sorry about that.